velox/substrait/VeloxToSubstraitPlan.cpp

```diff
@@ -365,6 +444,17 @@ ::substrait::Plan& VeloxToSubstraitPlanConvertor::addExtensionFunc(
   extensionFunction->set_function_anchor(7);
   extensionFunction->set_name("equal:i64_i64");
 
+  extensionFunction =
+      substraitPlan->add_extensions()->mutable_extension_function();
+  extensionFunction->set_extension_uri_reference(0);
+  extensionFunction->set_function_anchor(8);
+  extensionFunction->set_name("and:bool");
```

Comment:
We have discussed earlier that we should add proper support for function registry instead of continuing to hard-code functions. CC: @rui-mo

Reply:
Hi @mbasmanova. Can we unblock this PR right now before the function registry feature? Since a lot of work in my team are pending on this PR and we wish that it can be merged asap. 
Currently my team is working on the function registry part. We'll solve the hard code problem soon in the future.

Reply:
@kyotoyx I'm happy to hear your team is working on integration with function registry. This should simplify a lot of code and make it more robust. Let's proceed with a few more hard-coded functions for now then. Please, add a round-trip test for a query plan with join and ping for review.

Reply:
@kyotoyx 

> a lot of work in my team are pending on this PR

Would you share the project you are working on, so that I have a better understanding of the overall context for this PR?

Reply:
Hi @mbasmanova , we have 3 tasks WIP in parallel for substrait: `function registration`, `substrait upgrade to 0.7.0` and `join` as addressed in this PR. 
As for join task, will it be okay to have roundtrip join support in Velox first, together with [PR](https://github.com/facebookincubator/velox/pull/1675)? @rui-mo can provide an update against that PR. Of course, we will provide roundtrip test after both merged. 
In internal, our integration work is based on Velox and join support will allow related plan to parse, analyze and compute.

Reply:
@kyotoyx Let's discuss in Slack. This PR needs a test. It should be straightforward to add. See velox/substrait/tests/VeloxSubstraitRoundTripPlanConverterTest.cpp

Reply:
@mbasmanova Ok. I've sent an email to apply for the access to Slack. Could you please pull me in the Channel when it's convenient?

Reply:
CC: @jijufb

Reply:
@jijufb Hi could you please send me an invitation to your Slack channel/group?

Reply:
I believe @jijufb is on vacation. @pedroerp Pedro, do you know who else can help with Slack?

Reply:
@kyotoyx I can help you with this request. Can you please share your email Id so I can send you an invite? 

cc: @mbasmanova @pedroerp

Reply:
@sanumandla Thank you for all your assistance. My email is `yixiu.chen@intel.com`

Reply:
@kyotoyx can you please check if you got the invite.

Reply:
Hi @mbasmanova , my Team is now preparing the round-trip test case, I've sent the update information to you through Slack. :)

Reply:
> my Team is now preparing the round-trip test case

@kyotoyx Sounds great. Looking forward to an updated PR.